docs/docs/topics/original-request-context.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Original User Context
+description: This article describes how the original user context is passed secondary requests.
+---
+
+# Original User Context
+
+::: tip
+This article describes a use case available to [Pomerium Enterprise](/enterprise/about.md) customers.
+:::
+
+In enterprise environments where multiple services protected by Pomerium communicate with each other using a [service account](/enterprise/concepts.md#service-accounts), there are scenarios where the original user context must be preserved. This article describes how this is accomplished with the `X-Pomerium-Jwt-Assertion-For` header.
```

Comment:
we probably need have a page (and reference from here) how you can protect internal api resources with service accounts.

Reply:
Started in 4521af68cef3876712ee91e81879fc2ad0e92471